storage/aptosdb/src/pruner/state_store/mod.rs

```diff
@@ -118,11 +125,15 @@ impl StateMerklePruner {
     }
 
     /// Prunes the genesis state and saves the db alterations to the given change set
-    pub fn prune_genesis(state_merkle_db: Arc<DB>, change_set: &mut ChangeSet) -> Result<()> {
+    pub fn prune_genesis(
+        state_merkle_db: Arc<DB>,
+        ledger_db: Arc<DB>,
+        change_set: &mut ChangeSet,
+    ) -> Result<()> {
         let target_version = 1; // The genesis version is 0. Delete [0,1) (exclusive)
         let max_version = 1; // We should only be pruning a single version
 
-        let state_pruner = utils::create_state_pruner(state_merkle_db);
+        let state_pruner = utils::create_state_pruner(state_merkle_db, ledger_db);
         state_pruner.set_target_version(target_version);
 
         let min_readable_version = state_pruner.min_readable_version.load(Ordering::Relaxed);
```

Comment:
weird.. should just call state_pruner.prune_state_store(0, 1, 1) below, right?

Reply:
Unrelated to this PR. Will address in a separate PR.